frontend/src/organisms/QuackList.js

```diff
@@ -3,21 +3,18 @@ import React from 'react';
 import { Button, ErrorBanner, Loading } from '../atoms/';
 import { Quack } from '../molecules/';
 
-export function QuackList({ quacks, isLoading, error, onLikePress, refetch }) {
+export function QuackList({ quacks, isLoading, error, refetch }) {
   return (
     <>
       {isLoading && !quacks && <Loading />}
       {error && (
         <ErrorBanner title={error.message}>
-          <Button color="red" onClick={() => refetch()}>
+          <Button color="red" onClick={refetch}>
```

Comment:
Tohle neni to same. Kdyby nahodou `refetch` bral nejake optional argumenty, tak v tomto miste misto `refetch()` dostane `refetch(ClickeEvent)`.
Dej to prosim zpet.